pkg/template/servicebroker/servicebroker.go

```diff
@@ -0,0 +1,30 @@
+package servicebroker
+
+import (
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/client/cache"
+	"github.com/openshift/origin/pkg/controller/shared"
+	"github.com/openshift/origin/pkg/openservicebroker"
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+)
+
+type Broker struct {
+	kc                *kclientset.Clientset
+	oc                *client.Client
+	lister            cache.StoreToTemplateLister
+	templateNamespace string
+}
+
+func NewBroker(kc *kclientset.Clientset, oc *client.Client, informers shared.InformerFactory, templateNamespace string) *Broker {
+	return &Broker{kc: kc, oc: oc, lister: informers.Templates().Lister(), templateNamespace: templateNamespace}
+}
+
+// TODO: UUID this should be unique per cluster
```

Comment:
@smarterclayton @bparees  @deads2k @marun @derekwaynecarr I'm stuck on what the strategy should be for generating and storing a UUID for this that is different per installed cluster.